src/types/RokulRunnings.ts

```diff
@@ -21,6 +21,7 @@ export type XMLAttributes = {
   text?: string;
   error?: string;
   state?: string;
+  name?: string;
```

Comment:
any chance we could try to add all possible keys here? Makes it much easier to remember what I'm trying to reference
```
bounds: {x, y, width, height}
loadStatus: int
children: int
focusable: bool
extends: string
focusItem: int
color: colorString
visible: default true
opacity: default 100
uri:string
```

those are the ones I noticed right off the bat

Reply:
maybe event translate the bounds to the relative fields

Reply:
@slheavner Can you write this up as a feature request? It won't be just as simple as adding the fields, since we'll need to translate the strings that are returned into the relevant types...

For example, `visible` comes back as `"true"` or `"false"` and not `true`/`false`. That one would be easy but the bounds would probably be the most difficult.

Reply:
will do, happy to do this